config/rector.php

```diff
@@ -12,8 +12,7 @@
             __DIR__ . '/../tests',
         ]
     )
-    // uncomment to reach your current PHP version
-    // ->withPhpSets()
+    ->withPhpSets(php70: true)
```

Comment:
Is this really a problem? Because the CI tasks all run with PHP 8.3, and as soon as we hit the php72 set in a couple of PRs time, this parameter is altogether gone.

Reply:
But if you insist, I can switch this to
```PHP
    ->withSets([PHP_70])
```

Reply:
And done. Happy reviewing :)